### ISSUE TITLE
Website Responsiveness Enhancement

### ISSUE THREAD

**Issue by nandu-99** [khushi-joshi-05/Food-ordering-website#9]
Our open-source organization's website lacks responsiveness across various devices and screen sizes, leading to suboptimal user experience. Improving responsiveness is crucial for ensuring accessibility and usability for all users.Please assign this issue to me, i would love to resolve this issue

**Comment by JanitKumar** [comment]
I have knowledge about media queries. I think I can solve this issue. Assign it to me.

**Comment by suraj-9849** [comment]
I would like to work on this issue under @GSSOC'24 can you please assign me this issue!

**Comment by tat-tvam-asii** [comment]
@khushi-joshi-05 I can make this website responsive please assign this issue to me under GSSOC'24

**Comment by ShatakshiBansal** [comment]
@khushi-joshi-05 I have experience with making sites responsive. I would be glad if this issue is assigned to me

**Comment by Pujan-sarkar** [comment]
I want to contribute in this issue under GSSOC'24 and make this website full responsive . @khushi-joshi-05 will you please assign this issue to me .

**Comment by MelissaSequeira** [comment]
I can do it...Plz consider me for the task
https://melissasequeira.github.io/fashionweb.github.io/

**Comment by anglerfishlyy** [comment]
Mam , I worked with HTML , CSS and JS for a while now. I will be able to complete it . Assign me this issue under GSSOC'24.

**Comment by Khushinkm15** [comment]
i can do it by using media query so pls assign me this task

**Comment by aatmajajoshi** [comment]
i have knowledge about tailwind css and bootstrap of media queries. i can do this, please give me this issue, @nandu-99

**Comment by yash21sriv** [comment]
I want to work on that issue please assign me this issue

**Comment by bhuvangoel04** [comment]
Hi @khushi-joshi-05 , Can you please assign me this issue under GSSOC' 24, adding its level label too.

**Comment by Ketan787898** [comment]
I have a genuine interest in this project 
I have worked on our collage website also **done 2 web projects** you can cheak it on my repo
So ,Please assign this issue to me !! 

My project= https://github.com/Ketan787898/project-exibition

**Comment by anupa-rb** [comment]
Hi there, I'm a beginner at open-source contribution so I want to gain hands-on experience by contributing to this project through GSSOC'24. Please assign this issue to me. Thank you.
